eo-runtime/src/test/java/org/eolang/DataizedLogLevelTest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2022 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+import java.util.logging.Logger;
+
+/**
+ * Test case for dataization log level.
+ *
+ * @since 0.22
+ */
+@Execution(ExecutionMode.CONCURRENT)
+public final class DataizedLogLevelTest {
+    @Test
+    public void shortLogs() throws InterruptedException {
+        final Logger log = Logger.getLogger(Dataized.class.getName());
+        final Level before = log.getLevel();
+        log.setLevel(Level.ALL);
+        final List<LogRecord> logs = new LinkedList<>();
+        final Handler hnd = new Handler() {
+            @Override
+            public void publish(final LogRecord record) {
+                logs.add(record);
+            }
+
+            @Override
+            public void flush() {
+                throw new UnsupportedOperationException("#flush()");
+            }
+
+            @Override
+            public void close() throws SecurityException {
+                throw new UnsupportedOperationException("#close()");
+            }
+        };
+        log.addHandler(hnd);
+        Thread thread = new Thread(() -> {
```

Comment:
@Karina5005 Why do we need both `@Execution(CONCURRENT)` and new thread? I believe the annotation already creates new thread for the test so we can just set the property and create new `Dataized`.

Reply:
@mximp I found that it needs junit-platform.properties (https://stackoverflow.com/questions/50586201/parallel-test-case-execution-with-junit-5) to make it work. I didn't find such a file, so I added it (should I commit it or is it exist and I just did not find it?). But in parallel it still does not work the way I want, the contents of the test are parallelized. Therefore, I propose to leave the option with a separate thread.

Reply:
@Karina5005 Still why do we keep `@Execution` annotation if it doesn't have any effect. Can we leave only threads creation?

Reply:
@mximp yes, I removed annotation.

Reply:
@mximp I add `junit-platform.properties` and one test started to fall. It uses `@Execution(CONCURRENT)` and apparently before that the parallelism did not work and the test passed, but now it is falling. Should I just create an issue and wait for it to be fixed? So far, because of this, checkers do not pass.